r/RPlus/RPlus-4.2.2-foss-2022a-UPDio-1.1.0.eb

```diff
@@ -0,0 +1,413 @@
+#
+# This EasyBuild config file for RPlus was generated with generateEasyConfig.R
+#
+easyblock = 'Bundle'
+name = 'RPlus'
```

Comment:
Do not call this `RPlus`: will confuse users who are looking for the latest RPlus and expect > 1000 R packages.

Reply:
See discussion on Slack...